nb/pd-pandas-intro.ipynb

```diff
@@ -1622,34 +844,20 @@
     "on string content, just like Python string methods.  The `str` object of a `Series`\n",
     "(of a column of a data frame) is used to call string methods on each element, efficiently.\n",
     "The result is either a boolean `Series` that can then be used to retrieve rows from the data frame,\n",
-    "or a new string `Series` modified by the operation."
+    "or a new string `Series` modified by the operation. Something worthwhile mentioning here is that\n",
+    "we should convert the `object` data type to `Strig` for all columns that have String type data.\n",
+    "Using string methods on `object` data types still works, but is discouraged!"
```

Comment:
Needs an example instead.  Probably not worth making the example, the things at the bottom of this notebook are more worthy of being removed then anything else.  They're too short to be useful but too long to be boring enough.